lua/starfall/libs_cl/render.lua

```diff
@@ -2287,6 +2287,14 @@ function render_library.getScreenInfo(e)
 	return instance.Sanitize(screen.ScreenInfo)
 end
 
+--- Returns information about the current view setup.
+-- This does a table copy, so avoid using this in a draw hook for performance sake
+-- @param boolean? noPlayer If true, returns the view->GetViewSetup, if false - returns view->GetPlayerViewSetup. False by default.
```

Comment:
This second line of the doc isn't necessary. The noPlayer param isn't explained very well, what's it mean?